packages/build/src/core/build.ts

```diff
@@ -83,6 +83,7 @@ const tExecBuild = async function ({
   quiet,
   framework,
   explicitSecretKeys,
+  bootstrapURL,
```

Comment:
It feels weird to add yet another field to this long list, but it was the most straight-forward. Was wondering if maybe we should call it `edgeFunctionsBootstrapURL`, so it's less ambiguous?

Reply:
Yeah, I think that would help.

Reply:
Done in https://github.com/netlify/build/pull/5221/commits/683290a0e913fac58e555075180550fbcb652962.